app/components/Footer.tsx

```diff
@@ -0,0 +1,102 @@
+import {
+  Box,
+  Heading,
+  Grid,
+  GridItem,
+  Link,
+  LinkProps,
+  Stack,
+  Text,
+} from "@chakra-ui/react";
+import Image from "next/image";
+
+const links = [
+  { href: "/about-us", text: "About Us" },
+  { href: "/view-machines", text: "View Machines" },
+  { href: "/careers", text: "Careers" },
+  { href: "/request-quote", text: "Contact Us" },
+  { href: "/request-quote", text: "Request a Quote" },
+];
+
+const contactInfo = [
+  {
+    heading: "Mailing Address",
+    content: [
+      "14751 North Kelsey Street, Suite 105",
+      "PMB 616",
+      "Monroe, WA 98272-1458",
+    ],
+  },
+  {
+    heading: "Shipping Address",
+    content: ["30330 NE 172nd Street", "Duvall, WA 98019"],
+  },
+  {
+    heading: "Phone",
+    content: ["(425) 788-2100"],
+  },
+  {
+    heading: "Email",
+    content: ["sales@kenworthymachine.com"],
+  },
+];
+
+const Footer = () => {
+  return (
+    <Grid
+      templateColumns="repeat(5, 1fr)"
+      p={{ md: "42px 54px", lg: "72px 63px" }}
+    >
+      <GridItem colSpan={2}>
+        <Stack spacing={{ md: "50px", lg: "50px" }}>
+          <Box>
+            <Image
+              src="/images/kenworthy-logo.jpg"
+              alt="Kenworthy Machine"
+              width={500}
+              height={500}
+            />
+          </Box>
+          <>
+            {contactInfo.map(({ heading, content }) => (
+              <Box key={heading}>
+                <Heading variant="footer" pb={{ md: "15px" }}>
+                  {heading}
+                </Heading>
+                {content.map((line) => (
+                  <Text key={line} size={{ md: "l", lg: "xl" }}>
+                    {line}
+                  </Text>
+                ))}
+              </Box>
+            ))}
+          </>
+        </Stack>
+      </GridItem>
+      <GridItem colStart={4} colEnd={6}>
+        <Stack
+          spacing={{ md: "62px", lg: "57px" }}
+          pt={{ md: "8px", lg: "28px" }}
+        >
+          {links.map(({ href, text }) => (
+            <FooterNavLink key={text} href={href}>
+              {text}
+            </FooterNavLink>
+          ))}
+        </Stack>
+      </GridItem>
+    </Grid>
+  );
+};
+
+const FooterNavLink = ({ children, ...props }: LinkProps) => {
+  return (
+    <Box>
+      <Link variant="footer" size={{ md: "l", lg: "xl" }} {...props}>
```

Comment:
The navigation link text size also looks kind of large on my Macbook screen - but good on tablet. Is this an issue on just my screen or everyone's? If everyone's, maybe scale down the sizes in the size prop? 
<img width="1438" alt="Screenshot 2024-01-25 at 11 23 36 AM" src="https://github.com/wavecrestweb/kenworthy-machine/assets/115492619/4f11ddb3-ebe7-456f-86e4-f67e06d63083">

Reply:
@avinashi10 Yeah, that's something that I wanted to ask about: I've set up all the sizes including spacing exactly the way it appears on Figma. I agree that on large devices 36px for footer navigation is too much. But I wasn't sure if I needed to adjust the sizing from Figma using my best judgment or check with Kayla if she wanted me to stick to the sizing from Figma. As I haven't worked with designers and designs in Figma before, I would appreciate your thoughts on this.

Reply:
Archaa: All the sizes are correct according to Kayla's design (36px large, 32px medium), so Alex is spot on with every detail as far as I can tell.

Reply:
Sorry I should have clarified that my comment was based on what I visually thought was too big vs the wireframes. Great questions about translating wireframes Alex, I'm still pretty fuzzy on how to best do that. I do think there may be a missing piece where the figma wireframes can be built with CSS framework components to reduce guesswork, but Kayla built this before syncing up with us so maybe didn't specify. We should definitely ask her!

Reply:
I'm leaving font-sizes as is until Kayla updates the designs.